fjord/analytics/views.py

```diff
@@ -235,16 +227,16 @@ def unknown_to_empty(text):
     visible_products.append('')
 
     if search_product in visible_products:
-        f &= F(product=unknown_to_empty(search_product))
+        f &= F('term', product=unknown_to_empty(search_product))
         current_search['product'] = search_product
 
         if search_version is not None:
             # Note: We only filter on version if we're filtering on
             # product.
-            f &= F(version=unknown_to_empty(search_version))
+            f &= F('term', version=unknown_to_empty(search_version))
             current_search['version'] = search_version
     else:
-        f &= F(product__in=visible_products)
```

Comment:
In Elasticsearch 0.90, the "terms" query was aliased as "in":

https://www.elastic.co/guide/en/elasticsearch/reference/0.90/query-dsl-terms-query.html

I can't tell when they deprecated that alias, but it's not in the new Elasticsearch docs, so I switched it.